### ISSUE TITLE
make no_log: true configurable

### ISSUE THREAD

**Issue by thelittlebug** [bertvv/ansible-role-samba#53]
maybe it should be default to true, but it swallows every error

**Comment by achembarpu** [comment]
Agreed, I was facing issues with this role because of `no_log`. Could this be as simple as adding the output via `verbosity: N`?
